File: Lab-06/6_5.2/src/Entry.java
//Java program by Alex Bledsoe, 05/08/2017

public class Entry {
    private int k;
    private String v;

    Entry(int key, String value) {
        k = key;
        v = value;
    }

    int getKey() {
        return k;
    }

    public String toString() {
        return ("(" + k + ", " + v + ")");
    }
}

File: KotlinConversion/src/Lab_6/Num6_2/Entry.kt
package Lab_6.Num6_2

/**
 * Created by Alex Bledsoe, 6/11/2017.
 */
class Entry(key: Int, value: String) : Comparable<Entry> {
    var k: Int = 0
    var v: String? = null

    init {
        k = key
        v = value
    }

    override fun toString(): String {
        return "$v"
    }

    override fun compareTo(other: Entry): Int {
        return Integer.compare(this.k, other.k)
    }
}